package.json

```diff
@@ -85,5 +85,10 @@
     "node": ">=18.14.1"
   },
   "packageManager": "pnpm@9.1.0",
-  "prettier": "configs/prettier-config"
+  "prettier": "configs/prettier-config",
+  "pnpm": {
+    "overrides": {
+      "cookie@<0.7.0": ">=0.7.0"
```

Comment:
This needs to be under the `resolutions` field along with the rest of the overrides

Reply:
Yes, we should use either `pnpm.overrides` or `resolutions`, but not both. We can change `resolutions` to `pnpm.overrides` in a future PR if we want, but it's not important.